src/Evolution/Systems/GrMhd/ValenciaDivClean/PrimitiveFromConservative.cpp

```diff
@@ -180,10 +180,8 @@ bool PrimitiveFromConservative<OrderedListOfPrimitiveRecoverySchemes,
 
       // Check consistency
       if (use_hydro_optimization and
-          equal_within_roundoff(
-              tau[s] + rest_mass_density_times_lorentz_factor[s],
-              get(magnetic_field_squared)[s] * 0.5 + tau[s] +
-                  rest_mass_density_times_lorentz_factor[s])) {
+          (get(magnetic_field_squared)[s] <
+           100.0 * std::numeric_limits<double>::epsilon() * tau[s])) {
```

Comment:
@ermost is this the comparison I should be making? How large the b-energy is compared to the non-rest mass energy?

Reply:
I think this is fine. This should be a more conservative bound, since in the relevant regions rho*epsilon < rho.

Reply:
Okay, great, thank you!